packages/material-ui/src/ClickAwayListener/ClickAwayListener.js

```diff
@@ -85,9 +82,14 @@ class ClickAwayListener extends React.Component {
 
 ClickAwayListener.propTypes = {
   /**
-   * The wrapped element.
+   * @deprecated
```

Comment:
I guess we don't want to deprecate this yet. If the Fragment ref proposal is accepted we can simplify this component by removing the `getTargetEl` logic. Right now the Parent -> child relation isn't needed but it naturally emerges from the usage.